README.md

```diff
@@ -1,13 +1,32 @@
-# 🚧 THIS PROJECT IS IN ALPHA STAGE 🏗️
+# 💠 Emblem Giving
+---
+## 🚧 THIS PROJECT IS IN ALPHA STAGE 🏗️
```

Comment:
we should remove this alpha stage banner; I don't know what we are signalling with it now.